### ISSUE TITLE
Definition for rule '@typescript-eslint/consistent-type-assertions' was not found

### ISSUE THREAD

**Issue by elado** [jaredpalmer/tsdx#210]
### Current Behavior

Created a new `react` package, ran `lint src` and got an error:

```
$ npx tsdx create reactfoo
# choose "react"

$ cd reactfoo

$ yarn lint src

/tmp/reactfoo/src/index.tsx
  1:1  error  Definition for rule '@typescript-eslint/consistent-type-assertions' was not found  @typescript-eslint/consistent-type-assertions

✖ 1 problem (1 error, 0 warnings)

error Command failed with exit code 1.
```

### Expected behavior

no errors

### Suggested solution(s)

looks like this package doesn't exist anymore
https://www.npmjs.com/package/@typescript-eslint/consistent-type-assertions

when installing tsdx, it comes with `eslint-config-react-app@5.0.2` which has the reference to the missing package.
locking it at `5.0.1` resolves the issue.

looks like `6.0.2` is on `eslint-config-react-app` github but not yet released.

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| TSDX           | 0.9.2
| TypeScript       | 3.6.3
| Browser          |
| npm/Yarn         | yarn
| Operating System |

**Comment by jaredpalmer** [comment]
Can you send a PR?

**Comment by sadsa** [comment]
I've been tracking this same bug. It's a CRA issue - [See here](https://github.com/facebook/create-react-app/issues/7709)

**Comment by skvale** [comment]
I looked in the created `reactfoo/yarn.lock` file.  It looks like
```
tsdx@^0.9.3:
  ...
  dependencies:
    ...
    "@typescript-eslint/eslint-plugin" "^1.13.0"
    "@typescript-eslint/parser" "^1.13.0"
```
Which doesn't match the version needed to resolve `@typescript-eslint/consistent-type-assertions`.
I think tsdx just needs to publish a new version to fix this, since the right version is a dependencies now: https://github.com/jaredpalmer/tsdx/blob/master/package.json#L41

**Comment by Shelob9** [comment]
I found `yarn add eslint-config-react-app@5.0.1` to resolve it for me in my tsdx-generated project.

I can put in a PR, not sure what makes most sense given what @elado said:
>locking it at 5.0.1 resolves the issue.
>looks like 6.0.2 is on eslint-config-react-app github but not yet released.

Put in a PR to set 5.0.1 so it works now and then once 6.0.2 is out, a new PR for that?

**Comment by skvale** [comment]
As a different workaround for now, you could do `yarn add -D @typescript-eslint/eslint-plugin@latest`

**Comment by medelman17** [comment]
@skvale, @Shelob9 -- I believe this was closed by #220 and or #208.

**Comment by sadsa** [comment]
> @skvale, @Shelob9 -- I believe this was closed by #220 and or #208.

Yes, I believe this will be fixed by #216 - It's been merged into `master`. Now we're just waiting on release to `v9.0.4`

**Comment by agilgur5** [comment]
This was related to or fixed by #216 , #218, #220
